README.md

```diff
@@ -152,8 +152,8 @@ requests to the backend for execution and the backend uses the API to
 communicate with Triton.
 
 Every model must be associated with a backend. A model's backend is
-specified in the model's configuration using the 'backend' and
-'platform' settings. Depending on the backend one or the other of
+specified in the model's configuration using the 'backend'(recommended) and
+'platform'(legacy field supported for backward compatibility) settings. Depending on the backend one or the other of
```

Comment:
Instead of mentioning recommended / legacy field, perhaps just mention "For all backends, 'backend' must be set to the name of the backend and 'platform' is optional." (line 172-173) up here to be clear that which fields should be set. And perhaps adding "For backward compatibility reason, depending on the backend ..." to shows that the list below is the fewer exceptions.

Reply:
I like this. I think, in that case, you could get rid of the backend part of the below and shorten it substantially.

So, for example:

"For backward compatibility, the following backends allow you to optionally set the `platform` field instead of backend:

* For TensorRT, you can set the `platform` to *tensorrt_plan*.

* For PyTorch...."

@GuanLuo It's also been a while since we used the platform field, so maybe it's worthwhile to drop this documentation? The code still supports it for users already using platform, but perhaps we don't need to keep it documented? Especially given how easy it is to update the config.

Reply:
For me(as a new user to triton) it would be awesome to get rid of the backend part completely to not make it confusing. I would be happy to also update examples to make it be consistent through out the docs.

Reply:
I think we can definitely, prune the platforms field from our documentations and examples.